src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -1020,5 +1020,69 @@ public async Task When_VerticalContentAlignment_Is_Changed()
 			Assert.AreEqual(VerticalAlignment.Stretch, placeHolder.VerticalAlignment);
 			Assert.AreEqual(VerticalAlignment.Stretch, contentElement.VerticalAlignment);
 		}
+
+		[TestMethod]
+		public async Task When_Size_Zero_Fluent_Default()
+		{
+			var loaded = false;
+			using var styles = StyleHelper.UseFluentStyles();
+			var textBox = new TextBox
+			{
+				Text = "",
+				Width = 0,
+				Height = 0
+			};
+			textBox.Loaded += (s, e) => loaded = true;
+
+			WindowHelper.WindowContent = textBox;
+			await WindowHelper.WaitFor(() => loaded);
+
+			Assert.AreEqual(textBox.ActualWidth, textBox.MinWidth, 0.1);
+			Assert.AreEqual(textBox.ActualHeight, textBox.MinHeight, 0.1);
+		}
+
+		[TestMethod]
+		public async Task When_Size_Zero_Default()
+		{
+			var loaded = false;
+			var textBox = new TextBox
+			{
+				Text = "",
+				Width = 0,
+				Height = 0
+			};
+			textBox.Loaded += (s, e) => loaded = true;
+
+			WindowHelper.WindowContent = textBox;
+			await WindowHelper.WaitFor(() => loaded);
+
+			Assert.AreEqual(textBox.ActualWidth, 0);
+			Assert.AreEqual(textBox.ActualHeight, 0);
```

Comment:
It's weird, but the syntax for `Assert.AreEqual()` is `expected`, `actual`. It's one of the reason why I prefer FluentAssertions over the stock's Assert.AreEqual().

```suggestion
			Assert.AreEqual(0, textBox.ActualWidth);
			Assert.AreEqual(0, textBox.ActualHeight);
```